src/odemis/driver/test/actuator_test.py

```diff
@@ -1282,39 +1281,107 @@ def tearDown(self):
 STEP_SIZE = 5.9e-9
 
 CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
-CRYO_SECOM_CONFIG = CONFIG_PATH + "sim/cryosecom-sim.yaml"
+CRYO_SECOM_CONFIG = CONFIG_PATH + "sim/enzel-sim.yaml"
+
+KWARGS_5DOF = {
+    "name": "5DOF",
+    "role": None,
+    "ref_on_init": True,
+    "linear_speed": 0.01,  # m/s
+    "rotary_speed": 0.01,  # rad/s
+    "locator": "fake",
+    "hold_time": 0.1,  # s
+    "pos_deactive_after_ref": True,
+    "axes": {
+        'x': {
+            'range': [-3e-3, 3e-3],
+            'unit': 'm',
+        },
+        'y': {
+            'range': [-3e-3, 3e-3],
+            'unit': 'm',
+        },
+        'z': {
+            'range': [-1.e-2, 0.002],
+            'unit': 'm',
+        },
+        'rx': {
+            'range': [-0.785, 0.785],
+            'unit': 'rad',
+        },
+        'rz': {
+            'range': [-0.785, 0.785],
+            'unit': 'rad',
+        },
+    },
+}
+
+KWARGS_3DOF = {
+    "name": "3DOF",
+    "role": None,
+    "ref_on_init": True,
+    "locator": "fake",
+    "speed": 0.1,
+    "accel": 0.001,
+    "hold_time": 1.0,
+    "pos_deactive_after_ref": True,
+    "axes": {
+        'x': {
+            'range': [-3e-3, 3e-3],
+            'unit': 'm',
+            'channel': 0,
+        },
+        'y': {
+            'range': [-3e-3, 3e-3],
+            'unit': 'm',
+            'channel': 1,
+        },
+        'z': {
+            'range': [-0.02, 0.01],
+            'unit': 'm',
+            'channel': 2,
+        },
+    },
+}
+
+INIT_MD_3DOF = {
+    model.MD_FAV_POS_DEACTIVE: {"z":-2.e-3},
+}
+
+KWARGS_FOCUS = {
+    "name": "Focus",
+    "role": "focus",
+    "rng": [0, 4.2e-3],  # min/max positions in m
+}
+
+INIT_MD_FOCUS = {
+    model.MD_POS_COR: {'z':-0.003},
```

Comment:
Just a space needed here  `{'z':- 0.003}`

Reply:
Sorry, I won't do it, because Eclipse/PyDev doesn't allow me. It's part of its (broken?) auto formatting. I could work around it here, but I'm not ready to fight this fight in general.

Reply:
I will fix it with the added test case.